pages/linux/ddrescue.md

```diff
@@ -0,0 +1,11 @@
+# ddrescue
+
+> GNU ddrescue is a data recovery tool. It copies data from one file or block device (hard disc, cdrom, etc) to another, trying to rescue the good parts first in case of read errors.  https://www.gnu.org/software/ddrescue/.
```

Comment:
This looks like it's from the man page. It's rather verbose - what about this instead?
```suggestion
> Data recovery tool that reads data from damaged block devices.
> More information: <https://www.gnu.org/software/ddrescue/>.
```